src/EditorFeatures/VisualBasicTest/CodeActions/MoveType/MoveTypeTests.MoveToNewFile.vb

```diff
@@ -220,5 +220,44 @@ Partial Class Outer
 End Class"
             Await TestMoveTypeToNewFileAsync(code, codeAfterMove, expectedDocumentName, destinationDocumentText)
         End Function
+
+        <WorkItem(17879, "https://github.com/dotnet/roslyn/issues/17879")>
+        <WpfFact, Trait(Traits.Feature, Traits.Features.CodeActionsMoveType)>
+        Public Async Function DoNotMoveLeadingFileTrivia() As Task
+            Dim code =
+"
+''' <summary>
```

Comment:
Note: our general 'file banner' heuristic is:

```c#
<start-of-file> (whitespace* (single-comment|multi-comment) whitespace* newline)+ blankLine*
```

So doc-comments also do not count as file banners.